logging/console-logger.js

```diff
@@ -0,0 +1,16 @@
+// Log to the console
```

Comment:
I know you like giant files, but for my logging code I've split it up into class-based files. I'd prefer to keep it that way, since that's how my codebase has it, but if it's a problem they could be squashed.